rigex-front-end/src/components/WellModal/WellModal.js

```diff
@@ -173,10 +188,73 @@ export default function WellModal({
         <input
           name="longitude"
           type="text"
+          className="input-field"
           value={wellInfo.longitude}
           placeholder="Enter longitude"
           onChange={handleChange}
         />
+        <div className="input-label">
+          {' '}
+          Status<span>*</span>
+        </div>
+        <div className="status-row">
+          <div className="radio-container">
+            <img
+              alt="radio button"
+              className="radio"
+              onClick={() => {
+                const event = {
+                  target: {
+                    name: 'status',
+                    value: 'active',
+                  },
+                };
+                handleChange(event);
```

Comment:
it would be a bit more concise to write `onClick={() => handleChange({ target: { ... }})}`
not that what you wrote is wrong, you just don't need to set the object to a `const` before passing it through `handleChange` as an argument

Reply:
same feedback for the `onClick` on line 242

Reply:
and 223